.env.template

```diff
@@ -0,0 +1,23 @@
+# Authentication (DO NOT COMMIT)
+MULTISIG_ADDRESS=
+RPC_URL=
+PRIVATE_KEY=
```

Comment:
Should probably disambiguate between types of private keys, especially if we need to be scripting from different accounts (e.g., deployer vs. "multisig")

Reply:
Although actually does `forge script` just pickup PRIVATE_KEY automatically? Or is that still passed as CLI arg

Reply:
NM, I believe it does not (yet)

Reply:
Resolving convo w/ myself =)